### PR TITLE
THRIFT-5756 Run php tests in github actions

### PR DESCRIPTION
**Pull request by sveneld**
Before updating minimum version of php to 7.4 or 8.0 we should start running php tests on every pull request.
In this pull request i also maid such things:
- add xdebug to Docker instance ubuntu-bionic
- combine all php tests in one folder and add configuration file for running them
- add to github action running of tests for php versions 5.6 - 8.3
- for versions php greater than 7.1 in github action using dirty hack with update code of php tests, due to the BC in php
- modify README for better view

Next steps:
- increase code coverage for php code
- update min version of php to 7.4
- update generation of php code for 7.4

Also it will be great to update code style to PSR-12 and add code style check to github actions

- [X] Did you create an [Apache Jira](https://issues.apache.org/jira/projects/THRIFT/issues/) ticket? https://issues.apache.org/jira/browse/THRIFT-5756
- [X] If a ticket exists: Does your pull request title follow the pattern "THRIFT-NNNN: describe my issue"?
- [X] Did you squash your changes to a single commit?  (not required, but preferred)
- [X] Did you do your best to avoid breaking changes?  If one was needed, did you label the Jira ticket with "Breaking-Change"?
- [ ] If your change does not involve any code, include `[skip ci]` anywhere in the commit message to free up build resources.

### CODE REVIEW

**Comment by sveneld** [comment]
@pkvach

**Comment by pkvach** [comment]
@sveneld Great work, thank you.

**Comment by sveneld** [comment]
@Jens-G can you merge this pull request? After that I will continue with next step

File: lib/php/test/Unit/BaseValidatorTest.php [hunk 1]
```diff
@@ -114,27 +120,35 @@ protected static function assertHasReadValidator($class)
...
         }
     }
 
     protected static function assertNoReadValidator($class)
     {
         if (static::hasReadValidator($class)) {
             static::fail($class . ' class should not have a write validator');
+        } else {
+            static::assertTrue(true);
         }
```
Comment (Jens-G) [inline lib/php/test/Unit/BaseValidatorTest.php:134]: What's the value of this? A check that is known to always succeed?

Comment (sveneld) [reply lib/php/test/Unit/BaseValidatorTest.php:134]: if this check will be absent we will receive a warning that test does not make any assert, so if everything fine we should get a success assert

File: lib/php/phpunit.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (sveneld) [reply lib/php/phpunit.xml:1]: fixed

**Pull request merged**